tensorflow/stream_executor/dnn.h

```diff
@@ -190,6 +190,13 @@ class RnnDescriptor {
   virtual ParamsRegions ParamsBiasRegions() const { return ParamsRegions(); }
 };
 
+// Describes a CTC loss operation.
+class CtcLossDescriptor {
```

Comment:
I'm missing something here: why is this class empty (please add comments with the rationale).

Reply:
Maybe I'm misusing the github UI, but I still see is empty `CtcLossDescriptor` class.